vllm/envs.py

```diff
@@ -473,6 +474,10 @@ def get_default_config_root():
     # If set, use the V1 code path.
     "VLLM_USE_V1":
     lambda: bool(int(os.getenv("VLLM_USE_V1", "0"))),
+
+    # If set, enable multiprocessing in LLM for the V1 code path.
+    "VLLM_ENABLE_V1_MULTIPROCESSING":
+    lambda: bool(int(os.getenv("VLLM_ENABLE_V1_MULTIPROCESSING", "0"))),
```

Comment:
QQ: In which case should we turn this on?

Reply:
`VLLM_ENABLE_V1_MULTIPROCESSING=1` enables multiprocessing for `EngineCore` inside `LLM` (multiprocessing is used for `AsyncLLM`). It is faster than the current implementation.

<img width="788" alt="image" src="https://github.com/user-attachments/assets/f50b2cc9-a194-4c8e-9063-7648055177ab">

We will want to enable `VLLM_ENABLE_V1_MULTIPROCESSING=1`, but right now it is a problem for `LLM` since we cannot `spawn` without an `if __name__ == "__main__"` guard. We left solving this issue for follow up work.